core/src/main/java/org/mule/DefaultMuleMessage.java

```diff
@@ -1607,16 +1565,6 @@ private MuleMessage copyToInbound(DefaultMuleMessage currentMessage, Object payl
         return newMessage;
     }
 
-    void setSessionProperties(Map<String, TypedValue> sessionProperties)
-    {
-        properties.sessionMap = sessionProperties;
-    }
-
-    void setInvocationProperties(Map<String, TypedValue> invocationProperties)
```

Comment:
Not sure about this: before these changes, invocation properties were serialized with the message. If that is right: what happens now?